src/styles/page-card.scss

```diff
@@ -0,0 +1,93 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './stars';
+@import './colors';
+@import './card';
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+button,
+img {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  margin: 0;
+  height: 100vh;
+
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__reviews {
```

Comment:
Selector `.card__reviews` should be defined in `card.scss` since it's a BEM element of the `.card` block.